numpy/ma/core.pyi

```diff
@@ -229,6 +230,7 @@ _ScalarT = TypeVar("_ScalarT", bound=generic)
 _ScalarT_co = TypeVar("_ScalarT_co", bound=generic)
 # A subset of `MaskedArray` that can be parametrized w.r.t. `np.generic`
 _MaskedArray: TypeAlias = MaskedArray[_Shape, dtype[_ScalarT]]
+_MaskedArray1D: TypeAlias = MaskedArray[tuple[int], dtype[_ScalarT]]
```

Comment:
is this needed?

Reply:
not at the moment, will add it back in when it is